controllers/clusterlogging/clusterlogging_controller.go

```diff
@@ -58,7 +58,12 @@ var _ reconcile.Reconciler = &ReconcileClusterLogging{}
 type ReconcileClusterLogging struct {
 	// This Client, initialized using mgr.Client() above, is a split Client
 	// that reads objects from the cache and writes to the apiserver
-	Client   client.Client
+	Client client.Client
+
+	//Reader is an initialized client.Reader that reads objects directly from the apiserver
+	//instead of the cache. Useful for cases where need to read/write to a namespace other than
+	//the deployed namespace (e.g. openshift-config-managed)
```

Comment:
```suggestion
	// Reader is an initialized client.Reader that reads objects directly from the apiserver
	// instead of the cache. Useful for cases where need to read/write to a namespace other than
	// the deployed namespace (e.g. openshift-config-managed)
```